### PR TITLE
Feature/create tables

### PR DESCRIPTION
**Pull request by quantori-cherednichenko**

### CODE REVIEW

File: src/signals_notebook/entities/tables/table.py [hunk 1]
```diff
@@ -296,3 +299,100 @@ def get_html(self) -> str:
         log.info('Html template for %s:%s has been rendered.', self.__class__.__name__, self.eid)
 
         return template.render(name=self.name, table_head=table_head, rows=rows)
+
+    @classmethod
+    def creation(
```
Comment (quantori-pokidovea) [inline src/signals_notebook/entities/tables/table.py:304]: wat?

File: src/signals_notebook/entities/uploaded_resource.py [hunk 2]
```diff
@@ -29,7 +26,7 @@ def create(
         container: Container,
         name: str,
         content: bytes = b'',
-        content_type: str = ContentType,
+        content_type: str = 'application/octet-stream',
```
Comment (quantori-pokidovea) [inline src/signals_notebook/entities/uploaded_resource.py:29]: What happened?

Comment (quantori-cherednichenko) [reply src/signals_notebook/entities/uploaded_resource.py:29]: не может обратится к контенттайпу

File: src/signals_notebook/entities/tables/table.py [hunk 3]
```diff
@@ -268,6 +282,118 @@ def get(self, value: Union[str, UUID], default: Any = None) -> Union[Row, Any]:
...
             return default
 
+    @classmethod
+    def create(
+        cls,
+        *,
+        container: Container,
+        name: str,
+        content: bytes = b'',
+        content_type: str = ContentType.CSV.value,
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/entities/tables/table.py:292]: if we dont use .value property we will get this error while sending PATCH request
image

Comment (quantori-cherednichenko) [reply src/signals_notebook/entities/tables/table.py:292]: this bug appears in all Contentful entities while getting ContentType class for entity creation

Comment (quantori-cherednichenko) [reply src/signals_notebook/entities/tables/table.py:292]: In python 3.11 will appear StrEnum class from enum package.
This class will solve this problem

File: src/signals_notebook/entities/tables/table.py [hunk 4]
```diff
@@ -268,6 +279,119 @@ def get(self, value: Union[str, UUID], default: Any = None) -> Union[Row, Any]:
...
+                params={
+                    'digest': digest,
+                    'force': json.dumps(force),
+                },
+                json=request,
+            )
+            result = TableResponse(**response.json())
+            table = cast(ResponseData, result.data).body
+            log.debug('Entity: %s was created.', cls.__name__)
+            if content != b'':
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/entities/tables/table.py:334]: Do we need extra ContentType checking?

Comment (quantori-cherednichenko) [reply src/signals_notebook/entities/tables/table.py:334]: Or exclude options to add CSV file?

File: src/signals_notebook/entities/tables/table.py (hunk 3 above)
Comment (quantori-pokidovea) [reply src/signals_notebook/entities/tables/table.py:289]: OK

File: src/signals_notebook/entities/tables/table.py (hunk 4 above)
Comment (quantori-pokidovea) [reply src/signals_notebook/entities/tables/table.py:334]: Укажи в документации, что ожидается json

**Review by AndreyGuschinQuantori — APPROVED** [review]

**Review by quantori-pokidovea — APPROVED** [review]

**Pull request merged**